test/fixtures/no_preprocessor/input.ino

```diff
@@ -0,0 +1,13 @@
+// This should not be preprocessed
+
+#pragma PARTICLE_NO_PREPROCESSOR
+
+#include "Particle.h"
+
+int handler(int foo) {
```

Comment:
A good test here would be to include something only Particle.h would define, like `pinMode(D7, OUTPUT);` but also going beyond that test that the preprocessor is not running.  Is there a way to check for the absence of `int handler(int foo);` or maybe that `#line` does not exist?